test/server/index.test.ts

```diff
@@ -1,12 +1,81 @@
-import { C } from "./../../src/server";
+import request from "supertest";
+import { app } from "../../src/server/index";
 
-describe("Sample Test", function () {
-  it("should pass harmlessly", function () {
-    const c = new C();
+describe("App Server Tests", function () {
+  describe("Routes", function () {
+    it("should reject falsy routes and return 404 not found", function (done) {
+      request(app)
+        .get("/falsy/route")
+        .expect(404)
+        .expect("Content-type", "text/html; charset=utf-8")
+        .end(function (err) {
+          if (err) return done(err);
+          done();
+        });
+    });
 
-    expect(c.getX()).toEqual(10);
+    it("should return 200 OK for the id get route", function (done) {
```

Comment:
**Comment**: If the test title says "should return 200 OK", then we should also be testing the rest of the response. Here, we only know that the response bears status code of 200. Does the response body come back as expected?

Reply:
Merged server index tests with routes tests.